project/Dependencies.scala

```diff
@@ -73,6 +73,10 @@ object Dependencies {
     val logback = "ch.qos.logback" % "logback-classic" % "1.2.3" % "runtime" // Eclipse 1.0
   }
 
+  object Protobuf {
+    val googleCommonProtos = "com.thesamet.scalapb.common-protos" %% "proto-google-common-protos-scalapb_0.10" % "1.18.1-0" % "protobuf"
```

Comment:
Is there a need/way to deduct the versions (0.10, 1.18.1-0) from the protoc plugin?

Reply:
not AFAICS